WooCommerce/Classes/ViewRelated/Products/Edit Product/Inventory Settings/ProductInventoryEditableData.swift

```diff
@@ -1,34 +1,23 @@
 import Yosemite
 
-/// Contains editable properties of a Product in the inventory settings.
+/// Contains editable properties of a product model in the inventory settings.
 ///
-struct ProductInventoryEditableData {
+struct ProductInventoryEditableData: Equatable {
```

Comment:
Is the `Equatable` conformance missing, or I missed something?

Reply:
if not providing a custom `Equatable` implementation and all the properties already conform to `Equatable`, it will generate one internally based on all the properties! since our original implementation is just to check all the properties, it'd be safer for the struct to generate its own in case we add another property and forget to update the `Equatable` implementation 🙂